packages/ui-core/shared/src/lib/record-view/record-view-drawer.component.html

```
@@ -0,0 +1,34 @@
+	[attr.aria-hidden]="open ? null : 'true'"
+	[attr.aria-label]="heading | translate"
+>
+	<header class="record-drawer-header">
+		<div class="record-drawer-heading">
+			<h6 class="record-drawer-title">{{ heading | translate }}</h6>
+			@if (subtitle) {
+				<span class="record-drawer-subtitle">{{ subtitle }}</span>
+			}
+		</div>
+		<button
+			nbButton
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)